RELEASE-ANNOUNCEMENT.md

```diff
@@ -0,0 +1,42 @@
+We're excited to announce the availability of Jamulus 3.10.0!
+
+Some highlights include:
+- A new repository for Debian/Ubuntu users for automated upgrades. Please see the [Linux install page on the website](https://jamulus.io/wiki/Installation-for-Linux) for the setup process.
+- SRV-based virtual hosting support for improved user experience with servers running on non default ports.
+- An improved JSON-RPC API on the Server side to get more information about connected clients.
+
+Notes:
+- Please note that Jamulus 3.10.0 will from now on only support Windows 10 and 11 due to the upgrade to Qt 6.
+- The macOS legacy build is now considered deprecated and may be removed in future due to the lack of support for outdated Xcode versions on GitHub actions.
+- The CLI argument `--directoryserver` has been renamed to `--directoryaddress`. Please update your configuration.
+- If you compile Jamulus from source, please update the default branch to `main`. See [this announcement for more information](https://github.com/orgs/jamulussoftware/discussions/2984).
+
+Besides that, numerous usability improvements, enhancements, bugfixes and optimizations have been integrated.
+Please find all the details in the [Changelog](https://github.com/jamulussoftware/jamulus/releases/r3_10_0).
+
+## Downloads
+
+_Windows users: Please note that in the first days after release SmartScreen will probably display warnings about this release being unknown upon download and/or execution of the installer. Let us know when you do not see this warning anymore and we will update this announcement accordingly._
+
+**[↓ Windows](<!-- direct link to Windows version -->)** (ASIO version), alternative: [↓ JACK version](<!-- direct link to JACK version -->)
+**[↓ macOS (Universal)](<!-- direct link to macOS SIGNED Universal version -->)**
+**[↓ Debian/Ubuntu (amd64)](<!-- direct link to .deb [GUI] version -->)**, alternative: [↓ headless version](<!-- direct link to .deb [headless] version -->)
+**[↓ Debian/Ubuntu (armhf)](<!-- direct link to .deb [GUI] armhf version -->)**, alternative: [↓ headless version](<!-- direct link to .deb [headless] armhf version -->)
+**[↓ Debian/Ubuntu (arm64)](<!-- direct link to .deb [GUI] armhf version -->)**, alternative: [↓ headless version](<!-- direct link to .deb [headless] arm64 version -->)
```

Comment:
Maybe we should link to the install page for the repo instead of a direct download link here?

Reply:
Does that have highly visible download links for people who don't want to read about installing as they've been doing it for years?

Reply:
The Download links haven't been removed. The landing page (Jamulus.io) and every OS page still has clickable links.

Reply:
By "the installation page", what do you mean?  https://jamulus.io/wiki/Getting-Started ?  To get to the platform downloads for, e.g. iOS, it's not as easy as the current Release Announcement format, which links to the Github release page with _all_ the builds on.  That's really fast for existing users.

_Adding_ a link to "Getting Started" might not be a bad idea, for anyone who's not already read it several times.

Reply:
iOS and Android are that small for a reason. They should be considered experimental. If someone searches for them they'll find it.

Reply:
My comment was initially just for the Linux packets.

Reply:
> My comment was initially just for the Linux packets.

Ah right!  Yes, that might make it a lot more readable.

Reply:
I mean people should upgrade to the repository in my opinion. It's less work.

Reply:
In that case, changing the wording to the update instructions would be better (in future).  For this release, yes, directions on how to use the repo would be best.

Reply:
For now I've copied and condensed the stuff from the website, in future I'd add a link to the website and maybe link the general release page.